src/test/kotlin/it/pagopa/aca/service/AcaServiceTests.kt

```diff
@@ -168,11 +174,13 @@ class AcaServiceTests {
                     responseGetPosition,
                     requestCreatePosition,
                     iupd,
+                    ibanTestUpdate,
                     companyName
                 )
             )
         verify(gpdClient, Mockito.times(0)).createDebtPosition(any(), any())
         verify(gpdClient, Mockito.times(0))
             .invalidateDebtPosition(requestCreatePosition.paFiscalCode, iupd.value())
+        assertEquals(ibanTestUpdate, paymentPositionModelDtoCaptor.value.paymentOption?.get(0)?.transfer?.get(0)?.iban)
```

Comment:
```suggestion
        assertEquals(ibanTestUpdate, paymentPositionModelDtoCaptor.firstValue.paymentOption?.get(0)?.transfer?.get(0)?.iban)
```
and then here first value method from KArgumentMatcher class

Reply:
Did it!